### PR TITLE
property for bounds and tile coords

### PR DESCRIPTION
**Pull request by martham93**
adding property methods for a datapoint's bounds and tile coords

### CODE REVIEW

File: pyveda/training.py [hunk 1]
```diff
@@ -91,6 +91,14 @@ def id(self):
     def label(self):
         return self.data['label']
 
+    @property
+    def bounds(self):
+        return self.data['bounds']
+
+    @property
+    def tile_coords(self):
+        return self.data['tile_coords']
```
Comment (chelm) [inline pyveda/training.py]: So in some cases tile_coords wont be in data. Can you change this to something a bit safer? 

eg: self.data.get('tile_coords')

or you may need to check like:

```
if 'tile_coords' in self.data:
    return self.data['tile_coords']
else:
    return None
```

Comment (martham93) [reply pyveda/training.py]: makes sense thank you! I've added the check for tile coords

Comment (drwelby) [reply pyveda/training.py]: What about bounds for ungeoreferenced imagery? We haven't tackled that case yet but might want to be safe about that one too.

Comment (chelm) [reply pyveda/training.py]: yes both

**Pull request merged**